### PR TITLE
Feature fix login token and user info

### PR DESCRIPTION
**Pull request by jasondansie**
I now use the email retrieved from ldap to search the mongoDB for the logged in user. I then load the token with the data from the mongoDB data. I believe this should fix the problems you were having. 

I believe I when to every link and made sure it worked. If you find a link that does not work let me know.

### CODE REVIEW

**Review by CodeLaMat — DISMISSED** [review]
I changed the merging base from main to develop

**Review by fsuh — CHANGES_REQUESTED** [review]
the PR shows that there are some merge conflict between the branch and develop.  Please can you resolve the conflicts

**Review by fsuh — APPROVED** [review]

File: frontend/src/components/hr/surveys/HRSurveys.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const HRSurveys = () => {
   const navigate = useNavigate();
   const { t } = useTranslation();
   const userData = useAppSelector((state) => state.loginUser.userData);
-  const role = userData[0].role.join("");
+  const role = userData[0].role;
```
Comment (CodeLaMat) [inline frontend/src/components/hr/surveys/HRSurveys.tsx:29]: This format does not work for me.

**Review by CodeLaMat — APPROVED** [review]
I approve despite the codes that I find bit suspicious.

**Comment by jasondansie** [comment]
canceling this for now, will try again later.

**Pull request closed without merging**